sycl/include/CL/sycl/pipes.hpp

```diff
@@ -10,78 +10,12 @@
 
 #include <CL/__spirv/spirv_ops.hpp>
 #include <CL/__spirv/spirv_types.hpp>
+#include <CL/sycl/intel/pipes.hpp>
```

Comment:
No need for the other includes with the latest change

Reply:
Actually, these is a need of these include. Otherwise cl::sycl::intel::pipe isn't known at this point.

Reply:
I meant that all includes BUT this one can go

Reply:
Oh, yes, indeed.